app/views/auctions/_list_item.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="usa-grid">
     <div class='usa-width-one-whole issue-label'>
       <h3 class='issue-title'>
-        <span class="usa-label-big <%= auction_label_class(auction) %>"><%= auction_label(auction) %></span> <a href="<%= auction_path(auction) %>"><%= auction.title %></a>
+        <span class="usa-label-big <%= auction.label_class %>"><%= auction.label %></span> <a href="<%= auction_path(auction) %>"><%= auction.title %></a>
```

Comment:
I definitely get the dilemma: what would be the point of moving all that logic out of the model if you just had a really long Presenter? And so ViewModels (which are new to me) are indeed a useful approach for targeting methods to specific Controller views and avoiding the ugliness of `if-elsif-elsif-end` inside ERB files. But there are definitely some cases where we have to share the same presentation of attributes across multiple views and their corresponding API endpoints, and I guess those would have to be in a Presenter for now?